test/spec.zig

```diff
@@ -125,7 +125,7 @@ fn make(step: *Step, make_options: Step.MakeOptions) !void {
     const writer = output.writer();
     try writer.writeAll(preamble);
 
-    while (testcases.popOrNull()) |kv| {
+    while (testcases.pop()) |kv| {
```

Comment:
would be great if we could cherry pick this guy onto main? happy to pr that real quick if you're open to it! sorry for dropping in on pr, just noticed you clearly already saw this and figured id ask about it here rather than spam w/ a new issue.

Reply:
Sounds great!

Reply:
@carlmontanari would you like to the honours?

Reply:
Alright, I think I might beat you to it since I have managed to finish this branch now :-)

Reply:
😁 haha awesome! thanks a ton for all your work @kubkon !!